sdk/quantum/Azure.Quantum.Jobs/tests/QuantumJobClientLiveTests.cs

```diff
@@ -22,64 +24,147 @@ public QuantumJobClientLiveTests(bool isAsync) : base(isAsync)
 
         private QuantumJobClient CreateClient()
         {
+            // To be called only when recording locally
+            TestEnvironment.Initialize();
+
             var rawClient = new QuantumJobClient(TestEnvironment.SubscriptionId, TestEnvironment.ResourceGroup, TestEnvironment.WorkspaceName, TestEnvironment.Location, TestEnvironment.Credential, InstrumentClientOptions(new QuantumJobClientOptions()));
 
             return InstrumentClient(rawClient);
         }
 
         [RecordedTest]
-        public async Task CanGetList()
+        public async Task JobLifecycleTest()
         {
             var client = CreateClient();
 
-            int index = 0;
-            await foreach (JobDetails job in client.GetJobsAsync(CancellationToken.None))
+            // Get container Uri with SAS key
+            var containerName = "testcontainer";
+            var containerUri = (await client.GetStorageSasUriAsync(
+                new BlobDetails(containerName))).Value.SasUri;
+
+            // Create container if not exists (if not in Playback mode)
+            if (Mode != RecordedTestMode.Playback)
```

Comment:
This really isn't the right way to do it.  You should instead instrument your BlobContainerClient so it's recorded/played back correctly.  It doesn't have to be fixed before release, but it should be fixed soon.

Reply:
Actually, let's talk about this before you start it.  I don't really want you to have to record/manage Storage traffic.  I think there are patterns we could copy from other libraries here.